opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -127,6 +127,16 @@ namespace cpgrid
     {
         std::vector<std::size_t> removed_cells;
         if (ccobj_.rank() != 0 ) {
+            if(ecl_state) {
+                // Handle potential exception during MINPV processing
+                // Needed because later there is collective communication that will
+                // otherwise deadlock
+                int success = 1;
+                ccobj_.broadcast(&success, 1, 0);
+                if (success == 0) {
+                    throw std::runtime_error("Exception on master process during MINPV processing");
```

Comment:
The term "master process" is not precise and also not so useful for users. Suggest simply "Error encountered during MINPV processing".